src/vandermonde.jl

```diff
@@ -93,16 +115,17 @@ function Matrix(V::Transpose{T,Vandermonde{T}}) where T
     end
     M
 end
+=#
 
-function \(V::Adjoint{T1,Vandermonde{T1}}, y::AbstractVecOrMat{T2}) where T1 where T2
+function \(V::Adjoint{T1,Vandermonde{T1,C}}, y::AbstractVecOrMat{T2}) where {T1, C, T2}
```

Comment:
Since `C` is not used in the function body you could also just write
```suggestion
function \(V::Adjoint{T1,<:Vandermonde{T1}}, y::AbstractVecOrMat{T2}) where {T1, T2}
```